### DISCUSSION TITLE
2023/03/06/giscus/

### DISCUSSION THREAD

**Discussion by giscus** [BonyChops/blog#19] [Comment]
# 2023/03/06/giscus/

先日(というか昨日)，コメント機能としてutterancesを導入した．

https://blog-b2d8f--pr16-2023-03-05-giscus-n102zuse.web.app/2023/03/06/giscus/

**Comment by BonyChops** [comment]
コメントができるぞ！

**Comment by teletha** [comment]
### マークダウンでも書ける
[リンク](https://www.google.com/search) は不許可とかできるのかな？
